galera/src/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ target_compile_options(galera_smm
   )
 
 if (GALERA_VERSION_SCRIPT)
+if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
```

Comment:
agree, will fix

Reply:
> Know why https://github.com/freebsd/freebsd-ports/blob/main/databases/galera26/Makefile is unaffected?

I am not sure, but probably FreeBSD moved to LLVM linker in 12.0 (12.0 (December 11, 2018)) https://man.freebsd.org/cgi/man.cgi?query=ld&apropos=0&sektion=1&manpath=FreeBSD+12.0-RELEASE&arch=default&format=html since when they do fine and this patch https://github.com/codership/galera/commit/e907ec11201fbb9d89854fed87cec917c7e663c3 landed only in 2019. But I did not check to be sure.

Reply:
MacOS ld does not support this flag